src/org/omegat/core/search/Searcher.java

```diff
@@ -174,9 +174,15 @@ public void search() throws TranslationException, PatternSyntaxException, IOExce
         switch (expression.searchExpressionType) {
         case EXACT:
```

Comment:
It is not good for `REGEXP`, but good for `EXACT` and `KEYWORD`. A user who use `REGEXP` may want to control search behavior by itself. 
I'd like to make option disable when selecting `REGEXP`.